src/applications/vaos/services/appointment/transformers.js

```diff
@@ -354,3 +460,42 @@ export function transformConfirmedAppointments(appointments) {
     };
   });
 }
+
+/**
+ * Transforms VAR appointment request to FHIR appointment resource
+ *
+ * @export
+ * @param {Array} appointments An array of appointments from the VA facilities api
+ * @returns {Array} An array of FHIR Appointment resource
+ */
+export function transformPendingAppointments(requests) {
+  return requests.map(appt => {
+    const isCC = isCommunityCare(appt);
+
+    if (isCC) {
+      // CC requests to be handled in separate PR
+      return appt;
+    }
+
+    return {
+      resourceType: 'Appointment',
+      id: `var${appt.id}`,
+      status: getStatus(appt),
+      minutesDuration: 60,
+      type: {
+        coding: [{ code: appt.appointmentType }],
+      },
+      reason: getPurposeOfVisit(appt),
```

Comment:
Something new that's come up: there's also a reasonForVisit field in the request data that we'll need to show for express care requests. I think you could map that here.